shared/uavobjectdefinition/hwplayuavosd.xml

```diff
@@ -15,7 +15,7 @@
 		<field name="USB_HIDPort" units="function" type="enum" elements="1" options="USBTelemetry,RCTransmitter,Disabled" defaultvalue="USBTelemetry">
 			<description/>
 		</field>
-		<field name="USB_VCPPort" units="function" type="enum" elements="1" options="USBTelemetry,ComBridge,DebugConsole,Disabled" defaultvalue="Disabled">
+		<field name="USB_VCPPort" units="function" type="enum" elements="1" options="USBTelemetry,ComBridge,DebugConsole,Disabled" parent="HwShared.USB_VCPPort" defaultvalue="Disabled">
```

Comment:
You have removed the "options" attribute for the others, when you added the "parent" attribute.
Can't the "options" attribute also be removed here? The "parent" attribute already dictates these options, right?